kubernetes/pom.xml

```diff
@@ -29,4 +29,25 @@
     <module>ftest-kubernetes-resources</module>
   </modules>
 
+  <profiles>
+    <profile>
+      <id>jdk11-modular</id>
+        <activation>
+          <jdk>[17,)</jdk>
+        </activation>
+        <modules>
+          <module>fabric8</module>
+          <module>kubernetes</module>
+          <module>istio</module>
+          <module>reporter</module>
+          <module>ftest-kubernetes</module>
+          <module>ftest-kubernetes-logs</module>
+          <module>ftest-kubernetes-reporter</module>
+          <module>ftest-kubernetes-assistant</module>
+          <module>ftest-kubernetes-resources</module>
+          <module>ftest-kubernetes-wildfly-bootable-jar</module>
```

Comment:
I think we can only have this new module here, the other modules are enabled by default in any profile.

Reply:
Thanks, fixed!